src/activity.js

```diff
@@ -22,6 +22,12 @@ class Activity {
   calculateElevationClimbed() {
 
   }
+
```

Comment:
I agree with the move of this method to the activity class since it uses the data from activity in its calculations.  Along the same line on reasoning, I think we should move the two methods before it to here as well.